Formula/openfst.rb

```diff
@@ -5,6 +5,11 @@ class Openfst < Formula
   sha256 "9319aeb31d1e2950ae25449884e255cc2bc9dfaf987f601590763e61a10fbdde"
   license "Apache-2.0"
 
+  livecheck do
+    url "http://www.openfst.org/twiki/bin/view/FST/FstDownload"
+    regex /openfst-(\d(?:\.\d+)+).tar.gz/i
```

Comment:
```suggestion
    regex /href=.*?openfst[._-]v?(\d(?:\.\d+)+)\.t/i
```
Minor changes to match current standards.

Reply:
Just to note, we have RuboCops (with autocorrect!) in progress (Homebrew/brew#8643), so most of these changes should be easier to identify 😄